tests/python/frontend/onnx/test_forward.py

```diff
@@ -4872,6 +4872,48 @@ def test_qlinearadd():
     verify_qlinearadd([5, 1, 7], [2, 7], [5, 2, 7])
 
 
+def get_random_uniform(shape, dtype="float32", high=1.0, low=0.0, seed=None, target="llvm"):
+    ONNX_DTYPE = mapping.NP_TYPE_TO_TENSOR_TYPE[np.dtype(dtype)]
+    node = helper.make_node(
+        "RandomUniform", [], ["out"], shape=shape, dtype=ONNX_DTYPE, high=high, low=low
+    )
+    if seed is not None:
+        seed_attr = helper.make_attribute("seed", seed)
+        node.attribute.append(seed_attr)
+
+    graph = helper.make_graph(
+        [node],
+        "random_uniform_test",
+        inputs=[],
+        outputs=[helper.make_tensor_value_info("out", ONNX_DTYPE, shape)],
+    )
+    model = helper.make_model(graph, producer_name="random_uniform_test")
+    return get_tvm_output_with_vm(model, [], target=target, device=tvm.device(target, 0))
+
+
+def test_random_uniform():
```

Comment:
Can we also have a test which matches against a golden vector/tensor? 

I think actually testing for a uniform random distribution is really hard but having a golden vector/tensor at least shows to a user things look uniformly random for one seeded value.

Reply:
E.g. https://github.com/numpy/numpy/blob/main/numpy/random/tests/test_random.py#L950

Reply:
Yeah thats a great idea, thanks! I've added a test against a golden vector.